deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/StorageAuthenticationType.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.azure.sdk.iot.deps.serializer;
+
+import com.google.gson.annotations.SerializedName;
+/* Specifies authentication type being used for connecting to storage account */
+public enum StorageAuthenticationType
+{
+    /** Use a shared access key for authentication */
```

Comment:
This is missing the extra comment I added: This means authentication must be supplied in the storage URI(s)

Reply:
added